ruddock/modules/users/helpers.py

```diff
@@ -18,19 +36,32 @@ def get_memberlist(search_type):
     tables += " NATURAL JOIN members_current"
   elif search_type == "alumni":
     tables += " NATURAL JOIN members_alumni"
+  elif search_type in search_filters_grad_year:
+    pass
   else:
     # Something went wrong (this should have been validated earlier).
     raise ValueError
 
   # We also want usernames, or NULL if they don't exist.
   tables += " NATURAL LEFT JOIN users"
 
-  query = sqlalchemy.text("""
+  sqlText = """
     SELECT user_id, username, first_name, last_name, email,
       matriculation_year, graduation_year, membership_desc
     FROM {}
-    ORDER BY first_name
-    """.format(tables))
+    """
+
+  if search_type in ["all", "current", "alumni"]:
+    pass
+  elif search_type in search_filters_grad_year:
+    grad_year_str = search_type[4:8]
+    sqlText += "WHERE graduation_year=" + grad_year_str + " "
```

Comment:
Technically this isn't vulnerable to sql injection because of that "only if it's in the search_filters_grad_year list" clause up above, but it does spook me a bit.

Maybe just add a comment explaining why this is safe?